infra/gravity/testsuite.go

```diff
@@ -163,10 +163,16 @@ func (s *testSuite) getLogLink(testUID string) (string, error) {
 labels.__uuid__="%s"
 labels.__suite__="%s"
 severity>=INFO`, testUID, s.uid)},
-		}.Encode()}
+		}.Encode(),
+	}
 
-	short, err := s.client.Shorten(s.ctx, longUrl.String())
-	return short, trace.Wrap(err)
+	// Google UIRL shortener has been discontinued.
+	// See https://developers.googleblog.com/2018/03/transitioning-google-url-shortener.html for details.
+	// short, err := s.client.Shorten(s.ctx, longUrl.String())
```

Comment:
The service is in Google graveyard now so is there any reason to keep this code commented? I'd just keep the TODO item below.